recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "optuna" %}
-{% set version = "2.10.0" %}
+{% set version = "2.10.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 04c3100a4fe88a71dc72f69d8a10ec9db7c9a0c884feaf932b68632b00686a8d
+  sha256: 8a12009b57757c1070b3bff2261c24824d6430c22926dd1e2ace33b3deff555f
```

Comment:
The hash value seems to be correct.
https://pypi.org/project/optuna/2.10.1/#copy-hash-modal-94426bfe-6bb7-4652-8524-f7372063e304